from exceptions.base import BaseException


class NetworkError(BaseException):
    """
    Exception raised when Network class fails.
    """
